client/src/app/backend.service.ts

```diff
@@ -126,6 +136,13 @@ export class BackendService {
     return body;
   }
 
+  private configureSuccess(res:DataTransfer){
+    // Redirect to the export authorization flow after configure.
+    // this should be returned from the configure request and is checked
+    // upon creation of the DataTransfer object.
+    window.location.href=res.nextURL;
```

Comment:
Actually, I just checked and Angular has a location object you can import and use that is a bit more angular-ly
location.go("the url");

https://angular.io/api/common/Location
https://alligator.io/angular/location-service/

Reply:
So looking at this more, I think the location object is meant to control within the app. When I try location.go("https://external/oauth/url"); it prefixes it with the app domain (localhost/https://external/oauth/url). It also doesnt trigger a page reload to the new path (so having something valid like location.go("/next") only changes the address bar) 

The JS guide says to use the window.location instead for this: https://docs.angularjs.org/guide/$location